lib/light-service/organizer/scoped_reducable.rb

```diff
@@ -8,6 +8,14 @@ def scoped_reduce(organizer, ctx, steps)
 
         ctx
       end
+
+      def scoped_reduce_rollback(organizer, ctx, steps)
+        ctx.reset_skip_remaining! unless ctx.failure?
+        ctx = organizer.with(ctx).reduce_rollback(steps)
+        ctx.reset_skip_remaining! unless ctx.failure?
+
+        ctx
+      end
```

Comment:
I wonder if we can use this for other `orchestrator` methods. `reduce_if` and `reduce_until` would need this functionality as well.